amplipi/streams.py

```diff
@@ -1199,19 +1199,17 @@ def connect(self, src):
       # TODO: allow port to be specified with server (embedding it in the server URL does not work)
 
       self.proc = subprocess.Popen(args=lms_args)
-      self._connect(src)
-
-
+      self._connect(vsrc)
     except Exception as exc:
       print(f'error starting lms: {exc}')
 
   def _deactivate(self):
     if self._is_running():
       self.proc.kill()
-    self._disconnect()
-    if self.meta_proc is not None:
-      self.meta_proc.terminate()
-      self.meta_proc = None
+      self._disconnect()
```

Comment:
Please revert this change, disconnect should not be called directly by a PersistantStream.

Reply:
I don't know if meta_proc handling needs to be there, I'm trying to remember why it was removed